### ISSUE TITLE
[FEATURE] A Fixed Ostendo Manifesto

### ISSUE THREAD

**Issue by dojiks** [arkhometha/Historical-Project-Mod#211]
In HPM. Ostendo Manifesto never is obtained by most players due to the requirements. In history, the Ostendo Manifesto document was for pro-slavery Democrats to annex Cuba as a slave state and it was opposed everywhere that wasn’t the South.

The way to activate the decision in HPM is to have the reactionary party (the Know-Nothings) in power. 

America is a two-party system. The people in favor of this document were members of the Democratic Party not of any third-parties. 

My suggestion: Since this measure would have been based in Congress. Why not change the requirements to having the Democratic Party be in charge and that the reactionary party is 25% of the Upper House? This would mean that Democratic Party Senators with reactionary beliefs would be an influential part of Congress and not a minority and something as blatantly as pro-slavery as the annexation of Cuba (a colonial holding with slaves and a deeply entrenched slave culture so there was no question it was being admitted into the US as a slave state) to pass through congress and the Upper House.

**Comment by LoneCrusader2002** [comment]
I agree this decision needs work, however even a 25% Reactionary UH requirement seems to me hard to meet. (I've never been able to achieve a Reactionary government without "cheating" somehow, blast the incessant march of liberalism!) Maybe something more like a 75% Conservative UH.. I see what you're getting at referring to "Reactionary" Democratic senators, but "varying degrees" of intra-party ideology is not modeled in the game. Having a very high requirement for the Conservative party would allow them to lose northern/more liberal votes and still get such a bill passed...?

**Comment by dojiks** [comment]
> I agree this decision needs work, however even a 25% Reactionary UH requirement seems to me hard to meet. (I've never been able to achieve a Reactionary government without "cheating" somehow, blast the incessant march of liberalism!) Maybe something more like a 75% Conservative UH.. I see what you're getting at referring to "Reactionary" Democratic senators, but "varying degrees" of intra-party ideology is not modeled in the game. Having a very high requirement for the Conservative party would allow them to lose northern/more liberal votes and still get such a bill passed...?

I completely agree. Having 66% of Democratic Senators in the UH would mean that pro-slavery annexation attempts like Cuba wouldn't be halted in Congress via the Fillibuster/Cloture. Even Northern Democrats who were opposed to the expansion of slavery supported the acquisition of Cuba in their Party platform in 1860.

However, in HPM's start date...Conservatives make up 60% of the Upper House in 1836. The Annexation of Cuba was opposed because it was seen as expanding slave power and never brought up in Congress. I think the requirement saying 67% of the Upper House be conservative would be sufficient and attainable in a US game compared to having the Reactionary Party in Power as it currently stands. Hopefully, this can be fixed.
